<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelectProductConfigClass</class>
 <widget class="QDialog" name="DlgSelectProductConfigClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择产品配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="5,1">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gBox_dataDisplay">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbtn_ok">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_cancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_TextCurrentPath">
        <property name="text">
         <string>当前路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_filePath">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
